src/routing/types.ts

```diff
@@ -0,0 +1,13 @@
+export interface RouterProps {
+  children: Array<PageType>;
+  basename: string;
+}
+
+export interface PageType extends React.Component {
+  props: {
+    path: string; // Used by the Router directly
+    exact?: boolean;
+    children: React.Component;
```

Comment:
:thinking: Just one? Is there a type for one or more children?

Also, I think `React.Component` doesn't include native elements like `<div>`. Maybe use `JSX.Element` instead.

Reply:
Interesting...

In the test if I remove the second child div from a `Page` I get a complaint from typescript:

![image](https://user-images.githubusercontent.com/2008881/133169354-11182eb4-b7a0-4b39-b2a3-043b2bfa3e80.png)

Reply:
From the [TypeScript Handbook](https://www.typescriptlang.org/docs/handbook/jsx.html): 
![image](https://user-images.githubusercontent.com/12970166/133172142-49db4795-6363-4c26-b444-d660cf4ee991.png)

Reply:
That error makes me think that the `React.Component` type is just a `JSX.Element[]` behind the scenes, but I'll go for the union.